README.md

```diff
@@ -23,6 +24,19 @@ curl -X POST -H "Content-Type: application/json" --data "{}" http://localhost:88
 
 See the [spec](inst/schema/spec.md) for more details.
 
+## Updating data
+
+This will evolve as the upstream data changes and as our needs change. These scripts will create rds files that can be used to initialise the app. We'll store these either on mrcdata.dide.ic.ac.uk or as github release artefacts and pull them in fairly automatically during deployment.  They will expand into the actual mint database, which is much larger than the rds but faster to read.
```

Comment:
Not sure what "these scripts" refers to?

Reply:
this is out of date, will remove